woocommerce/woocommerce-ios
# [Mobile Payments] Add code to suppress Wallet during card capture

Closes #4079 

Changes:
- NOTE: I had to download the latest provisioning profile (July 8) for this to work.
- Adds the entitlement for wallet suppression for Debug, Release and Alpha
- Disables wallet when payment collection begins
- Restores wallet presentation when payment collection completed

To test:
- On a phone with Wallet set up (e.g. Apple Card or a bank debit or credit card)
- Start the Woo app
- Position the card reader directly next to the phone
- Enter an order and begin payment collection
- Ensure the phone's wallet does NOT appear
- Complete payment
- Launch the Woo app on another phone or tablet and bring the first device to the 2nd devices reader
- Ensure the first phone's wallet DOES appear. Cancel out of the wallet
- Repeat the above process, but instead of completing the payment cancel it and ensure the wallet DOES reappear when the 1st device is in proximity to the 2nd device's reader

Noteworthy:

The following appears on the screen the first time the merchant attempts to collect payment. This is expected behavior for this entitlement:

<img src="https://user-images.githubusercontent.com/1595739/125144806-e5058e80-e0d3-11eb-9bf7-b79d5c21c56f.PNG" width=50% />

cc @ctarda @astralbodies 

Update release notes:
